src/Files.App/Shell/Win32API.cs

```diff
@@ -428,16 +428,10 @@ public static IList<IconFileInfo> ExtractIconsFromDLL(string file)
             var iconsList = new List<IconFileInfo>();
             using var currentProc = Process.GetCurrentProcess();
             using var icoCnt = Shell32.ExtractIcon(currentProc.Handle, file, -1);
-            if (icoCnt == null)
-            {
-                return null;
-            }
+            if (icoCnt == null) return null;
```

Comment:
I think `return` should still be on the next line, just without the brackets.

Reply:
Done.